### ISSUE TITLE
mmctl: add role flag to users list

### ISSUE THREAD

**Issue by mkraft** [mattermost/mattermost#21197]
https://github.com/mattermost/mmctl/issues/558

**Comment by BabbarRaghav** [comment]
Hi, is this issue still open and without any assignees?

I would like to solve it

**Comment by hasnain003** [comment]
I would like to take this issue

**Comment by isacikgoz** [comment]
Hey @BabbarRaghav gentle ping on this issue. Are you willing to work on this? I couldn't find any associated PRs to this one. There are a couple of people wants to work on this, and it's okay if you can't find some time to spare this one.Just let us know!

**Comment by ChandanChainani** [comment]
@isacikgoz the issue seem already assigned to some here https://github.com/mattermost/mmctl/issues/558

**Comment by isacikgoz** [comment]
@ChandanChainani Yeah you are right, I want to wait a bit more until I got a response from @BabbarRaghav, otherwise I'll assign this to you. We generally track the assigned people here so, sorry for the confusion.

**Comment by isacikgoz** [comment]
@BabbarRaghav I'm unassigning you due to inactivity. Since @ChandanChainani was the first person actually asked to work on this, I'm assigning this to him.

**Comment by coderpilot** [comment]
Is there any movement on this?

**Comment by ChandanChainani** [comment]
@isacikgoz with recent version of mattermost server filter by roles in get users has been added.

Can I update [mattermost-server](https://github.com/mattermost/mmctl/blob/master/go.mod#L19) version to latest when released for implementation support for the feature?

**Comment by hanzei** [comment]
Making the ticket available again as https://github.com/mattermost/mmctl/pull/650 has been closed.

**Comment by annaos** [comment]
I would like to pick this.

**Comment by costa-neto** [comment]
Is this feature still needed? Can I work on it?

**Comment by hanzei** [comment]
Sure thing @costa-neto :+1:
